runbot_send_email/data/runbot_send_email_data.xml

```diff
@@ -172,7 +131,7 @@ Build #${object.sequence} is ${object.state} but failed
           </p>
         </span>
     </div>
-% elif (object.state=="running" or object.state=="done") and (object.result=="warn"):
+% elif object.result=="warn":
```

Comment:
what about ko result?

Reply:
@moylop260, it is the `else` in this [line](https://github.com/vauxoo-dev/runbot-addons/blob/bce30ed1bde6f846add1d6c14c4b5ae4a37bdd1e/runbot_send_email/data/runbot_send_email_data.xml#L90).